contentcuration/contentcuration/utils/publish.py

```diff
@@ -518,7 +519,7 @@ def process_formulas(content):
     return content
 
 
-def process_image_strings(content, zf):
+def process_image_strings(content, zf, channel_id):
```

Comment:
Ideally I wouldn't have done it this way, since ideally I'd have wanted to raise a specific exception that could have bubbled up and been logged with this context, but I wanted to avoid interrupting the current publishing flow, so I decided to just pass in the `channel_id` here.